src/bootstrap/src/core/build_steps/gcc.rs

```diff
@@ -210,7 +217,7 @@ fn build_gcc(metadata: &Meta, builder: &Builder<'_>, target: TargetSelection) {
         .env("CXXFLAGS", "-Wno-everything -g -O2")
         .env("CFLAGS", "-Wno-everything -g -O2")
```

Comment:
Is the `-Wno-everything` still necessary?

Reply:
I don't think so. Actually, I will add a check that someone doesn't try to compile GCC with Clang, similar to https://github.com/rust-lang/rust/pull/137457.